### ISSUE TITLE
Source Distribution tar missing for 0.10.18 on pypi.

### ISSUE THREAD

**Issue by kfreezen** [faust-streaming/faust#576]
The source distribution tar is missing for 0.10.18 on pypi.

I'm running a build process on an ARM64 Linux image in Docker, and no wheel is being built for that particular distribution.

Thanks!

**Comment by somnam** [comment]
Kindly asking for a fix:
```
pip install faust-streaming==0.10.18
ERROR: Could not find a version that satisfies the requirement faust-streaming==0.10.18
```

**Comment by wbarnha** [comment]
https://github.com/faust-streaming/faust/actions/runs/7008068061/job/19063802530

> removing 'faust-streaming-0.0.0' (and everything under it)

Man, I hate `setuptools`. Let me get this fixed for you.

**Comment by wbarnha** [comment]
Patched in `faust-streaming==0.10.19`, thank you all for your patience!
